components/providers/provider-schema-markup.tsx

```
@@ -0,0 +1,51 @@
+        "addressCountry": provider.country
+      }
+    }),
+    ...(provider.categories && provider.categories.length > 0 && {
+      "category": provider.categories.map(c => c.name).join(", ")
+    })
+  };
+
+  return (
+    <script
+      type="application/ld+json"
+      dangerouslySetInnerHTML={{ __html: JSON.stringify(schemaData) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)